wrappers/javascript/packages/aries-askar-nodejs/src/NodeJSAriesAskar.ts

```diff
@@ -321,9 +321,12 @@ export class NodeJSAriesAskar implements AriesAskar {
 
     const errorCode = this.nativeAriesAskar.askar_entry_list_get_value(entryListHandle, index, ret)
     this.handleError(errorCode)
-
     const byteBuffer = handleReturnPointer<ByteBufferType>(ret)
-    return new Uint8Array(secretBufferToBuffer(byteBuffer))
+    const view = secretBufferToBuffer(byteBuffer)
+    const result = Buffer.from(view)
+    this.nativeAriesAskar.askar_buffer_free(byteBuffer)
+
+    return new Uint8Array(result)
```

Comment:
Does .from copy the buffer? Is the behavior different from new Uint8Array?



How does it differ from:
```suggestion
    const result = new Uint8Array(secretBufferToBuffer(byteBuffer))
 this.nativeAriesAskar.askar_buffer_free(byteBuffer)

    return result
```